lib/rucio/core/dirac.py

```diff
@@ -67,12 +72,23 @@ def add_files(lfns, account, ignore_availability, vo='def', session=None):
     :param vo: The VO to act on
     :param session: The session used
     """
+    rule_extension_list = []
     attachments = []
     # The list of scopes is necessary for the extract_scope
     filter_ = {'scope': InternalScope(scope='*', vo=vo)}
     scopes = list_scopes(filter_=filter_, session=session)
     scopes = [scope.external for scope in scopes]
     exist_lfn = []
+    try:
+        lifetime_dict = config_get(section='dirac', option='lifetime', session=session)
```

Comment:
Please add this also to http://rucio.cern.ch/documentation/configuration_parameters
So it's documented there as well.